plugin/providers/keboola/resource_keboola_gooddata_user_management_v2_test.go

```diff
@@ -0,0 +1,61 @@
+package keboola
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccGoodDataUserManagement_v2_Basic(t *testing.T) {
```

Comment:
Pedantic I know, but function name should have `V2` instead of `v2` to match the other stuff that used `V2`.